test/spec/Feature/Query/ServerTiming.hs

```diff
@@ -0,0 +1,80 @@
+module Feature.Query.ServerTiming where
```

Comment:
It'd be better named as `ServerTimingSpec`, to maintain consistency with the other tests plus it's easier to find that way.